tests/test_htmlparser.py

```diff
@@ -0,0 +1,32 @@
+        <meta name="geo.placename" content="Pays de la Loire" />
+        <meta name="DC.box" content="name=Pays de la Loire; northlimit=48.567994064425; southlimit=46.26666162307; westlimit=-2.6257394729017; eastlimit=0.91665065791138; projection=EPSG3857" />
+        <meta name="ISO 19139" content="<gmd:EX_GeographicBoundingBox><gmd:westBoundLongitude><gco:Decimal>-2.6257394729017</gco:Decimal></gmd:westBoundLongitude><gmd:eastBoundLongitude><gco:Decimal>0.91665065791138</gco:Decimal></gmd:eastBoundLongitude><gmd:southBoundLatitude><gco:Decimal>46.26666162307</gco:Decimal></gmd:southBoundLatitude><gmd:northBoundLatitude><gco:Decimal>48.567994064425</gco:Decimal></gmd:northBoundLatitude></gmd:EX_GeographicBoundingBox>" />
+        """
+        self.soup = BeautifulSoup(html_doc, 'html.parser')
+        self.json_object = parse_html(self.soup)
+        self.assertEqual(self.json_object['type'], 'FeatureCollection')
+        self.assertEqual(self.json_object["features"][0]["geometry"]["type"], 'LineString')
+        self.assertIsNotNone(self.json_object)
```

Comment:
Add a test that parses a full article (take from the test files) and then

- saves the publication in the database
- check that the API now returns the publication and the information is correct

Reply:
Done in https://github.com/ifgi/optimetaPortal/pull/76/files#diff-34c98f76d2c694bcb7939bb20927517dc8a9745aeb9b7d855d7e1c9465ba54f0